packages/sync-actions/src/customers.js

```diff
@@ -44,6 +44,17 @@ function createCustomerMapActions(
       )
     )
 
+    allActions.push(
+      mapActionGroup('base', (): Array<UpdateAction> =>
```

Comment:
same action group so it wont be breaking change and its ordered after `addresses` actions